contracts/timealarms/src/alarms.rs

```diff
@@ -90,6 +99,60 @@ impl TimeAlarms {
             .is_some();
         Ok(AlarmsStatusResponse { remaining_alarms })
     }
+
+    fn try_add_old(
+        &self,
+        storage: &mut dyn Storage,
+        address: Addr,
+        time: TimeSeconds,
+    ) -> ContractResult<()> {
+        self.time_alarms
+            .add(storage, address, Timestamp::from_seconds(time))
+            .map_err(Into::into)
+    }
+}
+
+pub(super) struct InDelivery;
+
+impl InDelivery {
+    const ALARMS: Deque<'_, TimeAlarm> = Deque::new("alarms_in_delivery");
```

Comment:
please use a shorter namespace since it is multiplied in the DB a lot and contributes to high gas and high storage size